### PR TITLE
Update machine learning tutorial title and content clean up

### PR DESCRIPTION
**Pull request by krisdale**
- Update tutorial title to indicate what it will teach
- Simplify introduction
- Update CL name substitutions
- Update section title "Additional resources" to use standard "Related topics"
- Replace "Install and configure Clear Linux Host.." title with "Prerequisites". Update section to be more consistent with other prereq sections e.g. tutorial assumption and instruction to update CL.
- Correct section title delineation with correct depth

Signed-off-by: Kristal Dale <kristal.dale@intel.com>

### CODE REVIEW

File: source/clear-linux/tutorials/machine-learning/machine-learning.rst [hunk 1]
```diff
@@ -33,7 +40,7 @@ directory:
    sudo swupd bundle-add machine-learning-basic
```
Comment (mvincerx) [inline source/clear-linux/tutorials/machine-learning/machine-learning.rst]: Could you please code both commands here as `.. code-blocks:: bash`?

Comment (mvincerx) [reply source/clear-linux/tutorials/machine-learning/machine-learning.rst]: @krisdale Will you please change all other **input commands** to use same syntax as above in this document? 
`.. code-block:: bash `

**Comment by krisdale** [comment]
@mvincerx Good catch - updates applied and committed.

**Comment by mvincerx** [comment]
LGTM!

**Pull request merged**